### ISSUE TITLE
Indentação e itálico

### ISSUE THREAD

**Issue by hjbortol** [livro-aberto/BookCloud#378]
Me parece que não é um problema de indentação, mas de configuração de formatação deste tipo de nesting entre listas numeradas e legendas de figuras ...

![image](https://user-images.githubusercontent.com/7694624/35598782-60644d4c-060c-11e8-9952-e4833164def3.png)

**Comment by gutosurrex** [comment]
Qual é o problema exatamente? As legendas de outras figuras não ficam em itálico?

Outra pergunta, por que tanta indentação para colocar as figuras? Já tentou outras indentações?

**Comment by hjbortol** [comment]
Exatamente, Augusto. Nas figuras anteriores, as legendas estão sem itálico. Mesmo recuando a indentação, o problema do itálico continua! Obrigado!
